doc/user/optcarrot.md

```diff
@@ -0,0 +1,13 @@
+# Playing optcarrot
+
+Clone optcarrot repository.
+
+    git clone https://github.com/mame/optcarrot.git
+    cd optcarrot
+
+You will need mplayer to be installed, then you can play Lan Master 
+with following command.
+
+    path/to/graalvm-0.20/bin/ruby bin/optcarrot --video mplayer --input term examples/Lan_Master.nes
```

Comment:
Did you test it? I never tried with the upstream `mame/optcarrot`.
There might be issues for input.